### PR TITLE
Migrate AWS SDK for JavaScript v2 APIs to v3

### PR DESCRIPTION
**Pull request by trivikr**
# Description 📣

From AWS SDK for JavaScript v2 [README](https://github.com/aws/aws-sdk-js):
> We are formalizing our plans to make the Maintenance Announcement (Phase 2) for AWS SDK for JavaScript v2 in 2023.

This PR migrates AWS SDK for JavaScript v2 APIs to v3 using [aws-sdk-js-codemod](https://www.npmjs.com/package/aws-sdk-js-codemod).

```console
$ npx aws-sdk-js-codemod@0.26.2 -t v2-to-v3 backend/src/**/*.ts
```

## Type ✨

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation

# Tests 🛠️

```sh
# Here's some code block to paste some code snippets
```

---

- [ ] I have read the [contributing guide](https://infisical.com/docs/contributing/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/code-of-conduct). 📝

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@aws-sdk/client-ssm": "3.319.0",
```
Comment (kuhe) [inline backend/package.json:3]: should this be `"^3.319.0"?

Comment (trivikr) [reply backend/package.json:3]:
```suggestion
    "@aws-sdk/client-ssm": "^3.319.0",
```

**Comment by maidul98** [comment]
This PR is now stale. The backend structure has changed completely

**Comment by trivikr** [comment]
> This PR is now stale.

New PR posted at https://github.com/Infisical/infisical/pull/1413

**Pull request closed without merging**